scipy/stats/_continuous_distns.py

```diff
@@ -2209,6 +2303,12 @@ class truncweibull_min_gen(rv_continuous):
     def _argcheck(self, c, a, b):
         return (a >= 0.) & (b > a) & (c > 0.)
 
+    def _shape_info(self):
+        ic = _ShapeInfo("c", False, (0, np.inf), (False, False))
+        ia = _ShapeInfo("a", False, (0, np.inf), (True, False))
+        ib = _ShapeInfo("b", False, (0, np.inf), (False, False))
+        return [ic, ia, ib]
```

Comment:
This distribution doesn't appear in the rendered documentation!

Reply:
It was just recently merged. By me. I forgot that it's new. It's in the [dev documentation](http://scipy.github.io/devdocs/reference/generated/scipy.stats.truncweibull_min.html#scipy.stats.truncweibull_min).